UserScrape/app.py

```diff
@@ -5,8 +5,12 @@
 from data import load_all_seeds, seeds_for_user
 import asyncio
 
+
 cfg = load_cfg()
 video_seeds_df = load_all_seeds()
+repetitions = 2 # 100
+# todo: this list of videos needs to be sampled
+test_videos = ['uo9dAIQR3g8'] #, 'CH50zuS8DD0', '9_R3_CThc38']
```

Comment:
Can you sample these in the same way as you sampled the seed videos? The only difference is that this set should be the same for all accounts

Reply:
To be precise, this is how we wrote it down:
"This set of test videos is created by sampling one video from the top five channels of each category, again weighted by popularity. 
To have some baseline of (mostly) apolitical videos, each account also watches the 50 trending videos in the US of the day the experiment is run. The ‘test set’ consists therefore of 13*5 + 50 = 115 videos"
We could decide to also sample the channels themselves instead of taking the top 5. If we are sure that the top 5 most popular channels represent their class very well, I don't think we need to, but otherwise it might be a good idea.